src/pages/home/playlists/PlaylistList.tsx

```diff
@@ -0,0 +1,68 @@
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import MuiTableRow from "@mui/material/TableRow";
+import { useAppSelector } from "common";
+import {
+  TableCell,
+  TableCellImage,
+  TableHeadCell,
+  TableLine,
+  TableRow,
+} from "components";
+import { selectPlaylists } from "modules/playlist";
+import { FunctionComponent } from "react";
+import { useHistory } from "react-router-dom";
+
+const PlaylistList: FunctionComponent = () => {
+  const history = useHistory();
+
+  const playlists = useAppSelector(selectPlaylists);
+
+  return (
+    <TableContainer>
+      <Table>
+        <colgroup>
+          <col style={ { width: "15%" } } />
+        </colgroup>
+
+        <TableHead>
+          <MuiTableRow>
+            <TableHeadCell />
+            <TableHeadCell>Name</TableHeadCell>
+            <TableHeadCell>Songs</TableHeadCell>
```

Comment:
I think these headers are redundant. The name of the playlist is implied, and the number of songs is appended with the word "songs"

Not blocking, but maybe something to discuss with design

Reply:
Sounds good, definitely happy to discuss any changes with design.